cdap-security/src/main/java/io/cdap/cdap/security/authorization/DefaultAccessEnforcer.java

```diff
@@ -165,9 +169,10 @@ private void doEnforce(EntityId entity, Principal principal, Set<? extends Permi
   private void doEnforce(EntityType entityType, EntityId parentId, Principal principal, Permission permission)
```

Comment:
Please rename to `doEnforceOnParent` to be clear

Reply:
Done